src/components/transaction-item.tsx

```diff
@@ -121,14 +121,9 @@ const AddressArea = React.memo(
   }
 );
 
-const TxItemTitleArea = React.memo(({ title, isHovered, tx, principal }: any) => (
+const TxItemTitleArea = React.memo(({ title, tx, principal }: any) => (
```

Comment:
@kyranjamie what was the change you made in the wallet where we don't need to import `React` anymore? I think that would help clean up the explorer too.

Reply:
https://github.com/hirosystems/stacks-wallet-web/pull/2091

Reply:
@He1DAr I am going to make an issue to do this in the explorer.